modules/price/src/Form/CurrencyForm.php

```diff
@@ -53,6 +60,7 @@ public function form(array $form, FormStateInterface $form_state) {
     $form['currencyCode'] = [
       '#type' => 'textfield',
       '#title' => $this->t('Currency code'),
+      '#description' => $this->t('The uppercase alphabetic currency code. For example USD.'),
```

Comment:
Needs interpunction:
"For example: USD." 
(unsure if "For example: $example." is more common than "For example, $example.")